README.md

```diff
@@ -52,7 +52,7 @@ is not needed to install packages with these frameworks:
 | Decibel      | `decibel-app`
 | DokuWiki     | `dokuwiki-plugin`<br>`dokuwiki-template`
 | Dolibarr     | `dolibarr-module`
-| Drupal       | <b>`drupal-core`<br>`drupal-module`<br>`drupal-theme`</b><br>`drupal-library`<br>`drupal-profile`<br>`drupal-drush`
+| Drupal       | <b>`drupal-core`<br>`drupal-module`<br>`drupal-theme`</b><br>`drupal-library`<br>`drupal-profile`<br>`drupal-drush`<br>`custom-module`<br>`custom-theme`
```

Comment:
Maybe `drupal-custom-module` and `drupal-custom-theme`?

Reply:
I agree that would make more sense, but this pull request simply documents the values that already exist in code. Changing it at this point would be a backwards incompatible change and require a major version bump in order to follow semantic versioning.